### DISCUSSION TITLE
Allow non-dark mode

### DISCUSSION THREAD

**Discussion by treyhunner** [darrenburns/posting#3] [Ideas]
I expect all of my terminal apps to have the same background color and color scheme I'm already using.

I'd love it if this app either had a toggle between dark/light mode with a configurable setting *or* (even better IMO) if my OS's current dark/light setting could be sniffed out and used.

I've used the [darkdetect](https://pypi.org/project/darkdetect/) library [in another textual app](https://github.com/treyhunner/utf/blob/1cd864ce6f6fe7224e82af57a74ef987093e9fb0/utf/__init__.py#L5) to configure dark/light mode on startup per the current system setting. Maybe that could be used for implementing a similar feature for posting?

By the way, I love that this app exists! While demonstrating HTTP requests while teaching, I've been using [my own primitive app](https://github.com/treyhunner/http-tui) that does something similar and I'm excited to switch to yours. 👏

**Comment by darrenburns** [comment]
There probably won't be a dark/light toggle, but some of the themes available in the command palette will be light. I've still to plan the config out, so there may be autodetection, but I'm not sure :) You'll definitely be able to set the default theme using the config though.

https://github.com/darrenburns/posting/assets/5740731/071dedcc-52f2-467c-9888-813e24e89d98
